content/momentum/4/index.md

```diff
@@ -1,14 +1,16 @@
 ---
-lastUpdated: '03/26/2020'
-title: 'Momentum 4.x'
-description: 'Message Systems Inc Copyright 2014 Message Systems Inc Confidential Proprietary Abstract This book documents Momentum 4 Document generated on 2018 Aug 28 Table of Contents Preface 1 Typographical Conventions Used in This Document I Introduction to Momentum 1 Components 2 Life of A Message 3 Roles and Behaviors 4 Licensed...'
+lastUpdated: "03/26/2020"
```

Comment:
This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files